clippy.toml

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Ben Ford <ben.ford@kdab.com>
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+msrv = "1.78.0"
```

Comment:
maybe add a comment why this is higher than the other rust-version (because the clippy lint wasn't introduced until this version etc)